prepare-videochat.sh

```diff
@@ -287,27 +343,33 @@ if ! has_kernel_module v4l2loopback; then
     fi
 fi
 
+# Probe module if not probed yet
+if lsmod | grep v4l2loopback >/dev/null 2>/dev/null; then
+    # module is already loaded, do nothing
+    :
+elif [ $CAPTURE_STREAM = v -o $CAPTURE_STREAM = av ]; then
+    echo Loading module
+    sudo modprobe v4l2loopback #-q > /dev/null 2>&1
+fi
+
 # check if the user has the pulse gst plugin installed
-if find "/usr/lib/gstreamer-$GST_VER/libgstpulse.so" | egrep -q '.*'; then
+if find "/usr/lib/gstreamer-$GST_VER/libgstpulse.so" "/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstpulse.so" 2>/dev/null | egrep -q '.*'; then
```

Comment:
Perhaps we should use `/usr/lib/$(uname -a)-linux-gnu/gstreamer-$GST_VER/libgstpulse.so` instead?